cinema/serializers.py

```diff
@@ -30,10 +30,17 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
+
+
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = "__all__"
+        read_only_fields = ("image",)
```

Comment:
Better to be explicit about fields instead of __all__ . + you need to include image only on list and detail actions